src/style.css

```diff
@@ -0,0 +1,18 @@
+.element_1 {
+  background: red;
+  height: 100px;
+}
+
+.element_2 {
+  background: green;
+  height: 100px;
+}
+
+.element_3 {
+  background: blue;
+  height: 100px;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.